test/spanningtrees/kruskal.jl

```diff
@@ -40,3 +40,68 @@ push!(vec_mst, Edge(3, 4))
 push!(vec_mst, Edge(2, 3))
 
 @test mst == vec_mst
+
+#second test
+g2 = Graph(8)
```

Comment:
Don't override g2 here (it's defined in runtests.jl; we're debating how to move forward with it).